charts/rpe-service-auth-provider/values.template.yaml

```diff
@@ -0,0 +1,54 @@
+java:
+  applicationPort: 8489
+  environment:
+    TEST_SERVICE_SECRET : ${TEST_SERVICE_SECRET}
+    #JWTKEY : ${JWTKEY}
+    MICROSERVICEKEYS_CMC_LEGAL_FRONTEND: ${MICROSERVICEKEYS_CMC_LEGAL_FRONTEND}
```

Comment:
hmm is it worth going straight to using luigi's properties library? skip putting these in the environment variables?